src/pages/Components/RatingStars/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Rating Stars'
+order: 1
+---
+
+> Rating stars is a pattern you'll find in almost every website. It allows to do two different things. First, it allows the user to give a feedback on his experience with a product or a service. Second, it allows a brand to provide some informations on other user's opinion on a product or a service to make his choice easier.
+
+<preview path="src/pages/Components/RatingStars/previews/stars-input-basic" nude=true></preview>
+<preview path="src/pages/Components/RatingStars/previews/stars-result-basic" nude=true></preview>
+
+## Variations
+
+You can find three different versions of rating stars. One that represents empty stars, half stars and full stars.
+
+| Variation       | Usage                                                              |
+| --------------- | ------------------------------------------------------------------ |
+| Empty stars     | Empty stars are shown when the user hasn't posted a review yet     |
+| Half stars      | Half stars show an half rating of a user on a product or a service |
+| Full stars      | Full stars show user's opinion on a product or a service           |
+
+<preview path="src/pages/Components/RatingStars/previews/stars-result-all-states" nude=true></preview>
+
+### Sizes
+
+We designed rating stars with modularity in mind. You will find four different sizes of these to adjust to your needs.
+
+|                 |                                 |
+| --------------- | ------------------------------- |
+| **Small**       | <preview path="src/pages/Components/RatingStars/previews/stars-result-size-s" nude=true></preview> |
```

Comment:
no previews in tables please

Reply:
when we will do the responsive it will break, and when you change the size in the viewport selector it is not good

Reply:
I did the same we've done [there](https://patternratingstars-dot-demo-dot-mozaic-design-system-dtrp.appspot.com/Foundations/Typography/HeadingStyles/)

Reply:
yes but it is a mistake